src/Trixi2Img.jl

```diff
@@ -20,8 +20,11 @@ include("interpolation.jl")
 include("interpolate.jl")
 include("io.jl")
 
-# Include top-level convert method
-include("convert.jl")
+# Include top-level driver method
+include("trixi2img.jl")
```

Comment:
That's bad on Windows... You should rename that file to be usable on case-insensitive systems.

Reply:
Ugh, you're right. Any suggestions? `driver.jl`? `main.jl`?

Reply:
Good catch, by the way!

Reply:
`convert.jl`? `driver` or `main` are not really more descriptive, I think. But it's up to you - I don't have a strong opinion here.

Reply:
OK, I'll go back to `convert.jl`